### ISSUE TITLE
Mizrahi: Bank Transfers not including extra details

### ISSUE THREAD

**Issue by BackSlasher** [eshaham/israeli-bank-scrapers#784]
Assuming this transfer from a friend:
![image](https://user-images.githubusercontent.com/7194491/236489549-255fa4fb-7078-43cd-802e-2644249d3191.png)

The bank includes additonal details:
![image](https://user-images.githubusercontent.com/7194491/236489663-fc03b8b1-149d-4d11-81e8-0cce1d188433.png)

However the most we're getting via the scraper is:
```
{
  RecType: 1,
  IsTodayTransaction: false,
  RecTypeSpecified: true,
  Icone: ' 07',
  RowNumber: '8',
  TransactionNumber: '1',
  TotalRows: '22',
  Teur: null,
  NegdiBank: null,
  NegdiSnif: null,
  NegdiShem: null,
  NegdiCheshbon: null,
  TaarichEreh: null,
  TransactionType: 0,
  TransactionStatus: 0,
  RecordId5: null,
  MC02BlockCounter: null,
  MC02TnuaEZ: '01',
  MC02SugTnuaEZ: '1',
  MC02OfiTnuaEZ: '00',
  MC02PeulaTaaEZ: '2023-04-23T00:00:00',
  MC02PeulaTaaEZSpecified: true,
  MC02TnuaTeurEZ: '‏ב.הפועלים-ביט (י)',
  MC02PeulaShaaEZ: null,
  MC02AsmEZ: 99012,
  MC02AsmEZSpecified: true,
  MC02OfiSchumEZ: 1,
  MC02OfiSchumEZSpecified: true,
  MC02SchumEZ: 10.69,
  MC02SchumEZSpecified: true,
  MC02YitraEZ: 40392.66,
  MC02YitraEZSpecified: true,
  MC02ErehTaaEZ: '0001-01-01T00:00:00',
  MC02ErehTaaEZSpecified: true,
  MC02ShowDetailsEZ: '1',
  MC02KabahEZ: '116',
  MC02AsmahtaMekoritEZ: '000099012',
  MC02AgidEZ: '777',
  MC02KodGoremEZ: '968',
  MC02SugTnuaKaspitEZ: '324',
  MC02SeifMaralEZ: '96',
  MC02NoseMaralEZ: '0',
  kodMatbea: null,
  P428G2Details: null,
  P428G2Debit: null
}
```
Discovery of additional details in the website is done by calling out to `https://mto.mizrahi-tefahot.co.il/Online/api/OSH/getMaherBerurimSMF`, with params that can be translated like:
```
{
    "inKodGorem": source.MC02KodGoremEZ,
    "inAsmachta": source.MC02AsmahtaMekoritEZ,
    "inSchum": source.MC02SchumEZ,
    "inNakvanit": source.MC02KodGoremEZ,
    "inSugTnua": source.MC02SugTnuaKaspitEZ,
    "inAgid": source.MC02AgidEZ,
    "inTarPeulaFormatted": customFormat(source.MC02PeulaTaaEZ),
    "inTarErechFormatted": customFormat(source.MC02PeulaTaaEZ),
    "inKodNose": source.MC02SeifMaralEZ,
    "inTransactionNumber": source.TransactionNumber
  };
```

I'm trying to emulate the call from our code, but not sure about my approach
## TL;DR
We don't have metadata about bank wires, bit / paybox, etc

## What I want from you
1. Is this a problem to other people, or is it just me? Is someone already working on this and we can join forces?
2. Assuming we want this info, do we want to include it in some existing field (e.g.description) or a new one?
3. Is someone experienced with the Mizrahi code able to lend me a hand? I believe I'm missing some crucial info and therefore failing to fetch-post the requests correctly (getting blank responses instead of data).

Tanks for the great work :)

**Comment by esakal** [comment]
Thank you for your interest in the library. 

Unfortunately, as this library is not actively maintained, there is no roadmap or plans to introduce new features.

I personally don't have a Mizrahi account so I cannot help with that.

Your issue might remain open without any further interactions, however, there is a possibility that other users of this library might be interested in your request and engage with you.
